src/System.Net.Http/src/System/Net/Http/Headers/KnownHeaders.cs

```diff
@@ -11,81 +11,81 @@ internal static class KnownHeaders
     {
```

Comment:
There's an HPACK static table entry for the "Refresh" header, but we don't currently treat this as a known header. Seems like we should add this, for completeness.

Reply:
I looked it up when going through these but couldn't find any real official details on the header. Is it in an rfc?

Reply:
Here's the best info I can find: https://stackoverflow.com/questions/283752/refresh-http-header

We don't need to add any parsing support or anything like that, we can just add it to the KnownHeaders.